clients/manifest.go

```diff
@@ -21,6 +21,7 @@ import (
 const (
 	MasterManifestFilename = "index.m3u8"
 	DashManifestFilename   = "index.mpd"
+	ClipManifestFilename   = "clip.m3u8"
```

Comment:
@mjh1: for source playback to work correctly, thinking if this needs to be a `index.m3u8`?

Reply:
No that's all good 👍 tested it locally with the box and saw this in the master index.m3u8:
```#EXTM3U
#EXT-X-VERSION:3
#EXT-X-STREAM-INF:PROGRAM-ID=0,BANDWIDTH=4000000,RESOLUTION=640x360,NAME="360p"
//storage.googleapis.com/livepeer-test-vod/test-max/vod/2023-09-21T11:45:52.798Z/clip.m3u8
```